### ISSUE TITLE
MarkerStroke for scatter series not showing even though it is set

### ISSUE THREAD

**Issue by sushi19921992** [oxyplot/oxyplot#1607]
Hi all,

So I want to make the inside of the circle of my marker clear (using alpha var) and the stroke to show i.e. a green scatter point will have a green outer stroke and transparent inner.

I've tried the following:

    series.MarkerFill = OxyColor.FromArgb(1, series.MarkerFill.R, series.MarkerFill.G,
                    series.MarkerFill.B);

                series.MarkerStroke = OxyColor.FromArgb(0, series.MarkerFill.R, series.MarkerFill.G,
                    series.MarkerFill.B);
                series.MarkerStrokeThickness = 2;

But for some reason... the whole marker is coming up as transparent...

**Comment by VisualMelon** [comment]
The alpha value is a non-binary value, so you probably mean `255`, not `1`.

Also, by default, most everything has a default color of `OxyColors.Automatic`, which (in terms of RGB) is basically black: your method won't be picking up the color that is assigned automatically (this is all resolved at render time)

**Comment by sushi19921992** [comment]
Doodoo..... Any suggestions on what I can do to get the inner circle empty but have the stroke around it be assigned the colour at render time?

**Comment by VisualMelon** [comment]
No, it doesn't look like it. There was some discussion recently about addressing this, but I can't seem to find it...

**Comment by sushi19921992** [comment]
Hmm yeah that does kinda suck as that should definitely be something you can change without the colours being set to black...
